src/app/opengraph-image.tsx

```
@@ -1,49 +1,49 @@
+                    alignItems: "center",
+                    justifyContent: "center",
+                    gap: "60px",
+                    padding: "60px",
+                }}
+            >
+                <img
+                    src={photoBase64}
+                    width={200}
+                    height={200}
+                    style={{borderRadius: "100px", objectFit: "cover"}}
+                />
```

Biome (2.4.11)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

(lint/a11y/useAltText)